src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -57,6 +57,8 @@ public ReadOnlyPerson getPerson() {
         return toAdd;
     }
 
+    public boolean isMutating(){ return true; }
```

Comment:
You should explicitly stated this is an overiding method, and if it behaves different from its parent, then a header comment is required as well. This helps increasing readability of your code.